openshift-template.yaml

```diff
@@ -112,6 +112,10 @@ parameters:
     type: boolean
     description: download certificates with certbot during install
     default: false
+  do_upgrades:
+    type: boolean
+    description: Whether to perform upgrade as part of Ansible deployment
+    default: false
```

Comment:
I think this should default to true personally - better to accidentally patch to latest than to deploy on older packages? Then we have to choose not to if we want faster tests.